plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeConfig.java

```diff
@@ -43,6 +43,7 @@
     private DataSize exchangeSinkMaxFileSize = DataSize.of(1, GIGABYTE);
     private int exchangeSourceConcurrentReaders = 4;
     private int maxOutputPartitionCount = 50;
+    private int exchangeFileListingParallelism = 100;
```

Comment:
I'm a little afraid that this value may be too much. Could you please run a full scale testing at least once with this patch included?

Reply:
Sure